### PR TITLE
Updated Documentation for Education Module

### LINKED ISSUE #4374: RFE: reeducation camp documentation clarifications
Personnel Module documenation referencing reeducation camps: 

- Turnover & Retention Module document refers to the Education Module for more information regarding reeducation camps (pg. 11). 

- Education Module document does not actually have a single heading explaining what a reeducation camp does. It refers piecemeal to seperate parts of the reeducation camp functionality, but does not provide a single overview of what this does, or how it is meant to interact. 

-> Create a single entry in Education Module that covers all these loose elements and explains interaction with Turnover & Retention module under one heading.

### PR DESCRIPTION
**Pull request by IllianiCBT**
This PR updates the Education Module documentation to include all new features and adjustments.

Closes #4374

### CODE REVIEW

**Review by HammerGS — APPROVED** [review]

**Comment by IllianiCBT** [comment]
Holding off merging this until #4382 has been merged, as the updated documentation references changes made in that PR.

**Pull request merged**
